contrib/packs/actions/install.meta.yaml

```diff
@@ -21,5 +21,5 @@
       default: "master"
     register:
       type: "string"
-      default: "actions"
+      default: "actions,aliases,sensors"
```

Comment:
For now if you have figured out a way we should go with that since we know that it works.